### DISCUSSION TITLE
Form Data Flow

### DISCUSSION THREAD

**Discussion by jali-k** [cepdnaclk/e19-co227-ACES-Project-Fund-Requests#14] [Announcements]
I've implemented sending data collected with formsection1, under the POST rout of `/contactDetails`. @sayumimuthu, you can use that data to send to the authorities UI from the backend.

**Comment by jali-k** [comment]
Form section two (About the project) is also sending the data to the backend now `/aboutProject`. @sayumimuthu 🙂

**Comment by sayumimuthu** [comment]
@mewanDimalsha Can you give an update on the database?

**Comment by sayumimuthu** [comment]
Added a new function as 'sendToAdmin' to send the contactDetails to admin. Need to update the adminEndpoint URL in that function. Can you check on its functionality? Then can add the function on other post routes as well to send the relevant data to the admin. @jali-k

**Reply by jali-k** [reply]
I explained you! Please try 🙂

**Comment by jali-k** [comment]
Created a get request on `/find/:id` that you can pass an id there and, find the related request from the database. @sayumimuthu You can try to send that data to the admin page(without styling for now) and show them there.
- Add a few requests using students' font-end
- Use their ids to take the data from the database
- Show them in admin page

**Comment by jali-k** [comment]
With `/getall` rout I take the all documents in the database to the frontend. You can view it through `inspect -> console` in google chrome. Try to show cards as per those document and implement `view more` functionality.  @sayumimuthu @mewanDimalsha
